subprojects/docs/src/main/resources/header.html

```diff
@@ -88,81 +88,66 @@
             <input type="search" name="q" id="search-input" class="search-input" placeholder="Search the docs"/>
         </div>
         <ul>
-            <li class="docs-home-link"><a href="/userguide/userguide.html">Gradle Overview</a> <div class="pdf-link-container">
+            <li class="docs-home-link"><a href="/userguide/userguide.html">Documentation Home</a> <div class="pdf-link-container">
```

Comment:
I'm OK with that, but I do think the PDF link should be next to "User Manual" rather than here. Otherwise it's not clear what the PDF link will give you.

Reply:
Right. Agreed.